client/components/header-cake/back.jsx

```diff
@@ -39,7 +41,7 @@ export default React.createClass( {
 
 	render() {
 		const { text = this.translate( 'Back' ), href, onClick, spacer } = this.props;
-		const windowWidth = window.innerWidth;
+		const windowWidth = global.window.innerWidth || 1024;
```

Comment:
Thanks for the explanation. :)

May be good to keep it as a constant where we can add a comment to justify it. Otherwise in a few months we may wonder if we had good heuristics to arrive at 1024.